ctl/version/version.go

```diff
@@ -49,47 +53,83 @@ kmeshctl version <kmesh-daemon-pod>`,
 
 // RunVersion provides the version info of kmeshctl or specific Kmesh daemon.
 func RunVersion(cmd *cobra.Command, args []string) error {
-	if len(args) == 0 {
-		v := version.Get()
-		cmd.Printf("%s\n", v.GitVersion)
-
-		return nil
-	}
-
 	cli, err := utils.CreateKubeClient()
 	if err != nil {
-		log.Errorf("failed to create cli client: %v", err)
+		log.Errorf("failed to create kube client: %v", err)
 		os.Exit(1)
 	}
 
+	cli.Kube()
+
+	if len(args) == 0 {
+		v := version.Get()
+		cmd.Printf("client version: %s-%s\n", v.GitVersion, v.GitCommit)
+		podList, err := cli.PodsForSelector(context.TODO(), utils.KmeshNamespace, utils.KmeshLabel)
+		if err != nil {
+			log.Errorf("failed to get kmesh daemon pods: %v", err)
+			os.Exit(1)
```

Comment:
If use `os.Exit(1)`, I think is it better to change the function from return error to no return value.

Reply:
make sense